Release v9.2.0 (#1373)
newrelic/node-newrelic: NEWS.md

### v9.2.0 (2022-10-06)

* Added ability to instrument ES Modules with the New Relic ESM Loader.
  * [Example ESM application](https://github.com/newrelic/newrelic-node-examples/tree/main/esm-app)

* Added support for custom ESM instrumentation.
  * There is structure to registering custom ESM instrumentation.  Set the relative path to the instrumentation entry point via `api.esm.custom_instrumentation_entrypoint`

>   * [Sample custom ESM instrumentaiton entrypoint](https://github.com/newrelic/newrelic-node-examples/blob/main/esm-app/custom-instrumentation/index.js)

Reviewer: ```suggestion
  * [Sample custom ESM instrumentation entrypoint](https://github.com/newrelic/newrelic-node-examples/blob/main/esm-app/custom-instrumentation/index.js)
```

>   * All the `newrelic.instrument*` methods will still work except `newrelic.instrumentLoadedModule`.  This is because it was geared toward CommonJS modules. 

Reviewer 2: ```suggestion
  * All the `newrelic.instrument*` methods will still work except `newrelic.instrumentLoadedModule`.  This is because it is geared toward CommonJS modules. 
```

still geared

* Added test for asserting ESM loader functionality on ESM-only package

* Added supportability metric of `Supportability/Nodejs/Collector/MaxPayloadSizeLimit/<endpoint>` when `max_payload_size_in_bytes` configuration value is exceeded.

> * Removed `application_logging.forwarding.enabled` stanza from sample config as the feature is Generally Available.

Reviewer 2: ```suggestion
* Removed `application_logging.forwarding.enabled` stanza from sample config as the feature is now enabled by default.
```

"GA" is a bit too jargony, even when expanded out with capital letters.

Reviewer 3: It's a common term in tech. I spelled it out too. I can change but this jargon leaks throughout our whole org
